include/dd/ComplexNumbers.hpp

```diff
@@ -150,6 +150,11 @@ namespace dd {
             auto vali = CTEntry::val(c.i);
             return lookup(valr, vali);
         }
+        Complex lookup(const std::complex<double>& c) {
+            auto valr = c.real();
+            auto vali = c.imag();
+            return lookup(valr, vali);
```

Comment:
```suggestion
            return lookup(c.real(), c.imag());
```
No need to waste more lines than necessary here as the local variables are not used anywhere besides the return value.